.github/workflows/rust-test.yml

```diff
@@ -107,8 +111,18 @@ jobs:
           - type: miri
             toolchain: ${{ inputs.nightly_toolchain }}
       fail-fast: ${{ inputs.fail_fast == 'true' }}
+    services:
+      azurite:
+        # Hack for conditionally enabling the service:
+        # https://github.com/actions/runner/issues/822
+        image: ${{ (inputs.service_azurite) && 'mcr.microsoft.com/azure-storage/azurite' || '' }}
```

Comment:
This is awesome

Reply:
Would be more awesome as an "if" like jobs have.